.github/dependabot.yml

```diff
@@ -0,0 +1,6 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: ".github/workflows"
+    schedule:
+      interval: "weekly"
```

Comment:
What is this file doing exactly? I see that the file is called `dependabot.yml`, but what is it checking weekly? The lychee-action version? We can simply not fix it, no? I wouldn't see much harm in this situation.

Reply:
This file informs dependabot to check every week if the GitHub Actions used in the workflows have updates.
When a new version exists dependabot will open a pull request with the update.

Example: https://github.com/lycheeverse/lychee-action/pull/98

The PR can be closed/ignore I think (never maintained a repo with dependabot).

lychee and lychee-action are evolving rapidly and recent features made a difference to me to open this PR. Since the action is really focused on this repo use case, validate documentation links, new versions are probably going to be better.

Reply:
What I meant is that instead of `@v1.4.1` we could use `@v1`, and avoid this. Unless your point is to know what the upgrades are!

Reply:
Good point, I always forget that I can use semantic version :D

At this moment I think it's best to know what the upgrades are because I expect/hope that future versions to support more proper action arguments to avoid using the `args:` with so much arguments which requires to known more about lychee.

Reply:
I mean, if it is breaking changes, they should bump the major, but sure, I don't really think it matters to much.